thirdeye/thirdeye-frontend/app/pods/components/rootcause-dimensions-table/component.js

```diff
@@ -0,0 +1,265 @@
+import { computed, getProperties } from '@ember/object';
+import Component from '@ember/component';
+import {
+  toCurrentUrn,
+  toBaselineUrn,
+  isInverse,
+  toColorDirection,
+  makeSortable,
+  appendFilters,
+  hasPrefix,
+  isAdditive
+} from 'thirdeye-frontend/utils/rca-utils';
+import {
+  humanizeChange,
+  humanizeFloat
+} from 'thirdeye-frontend/utils/utils';
+import _ from 'lodash';
+
+const ROOTCAUSE_TRUNCATION_FRACTION = 0.0001;
+
+export default Component.extend({
+  classNames: ['rootcause-metrics'],
+
+  /**
+   * Columns for dimensions table
+   * @type Object[]
+   */
+  dimensionsTableColumns: [
```

Comment:
Hi Alex. Can you please put this under `shared` folder like that of `app/shared/anomaliesTableColumns.js`? It would be cleaner for this file. Thanks!

Reply:
fixed